tests/test_task_details.py

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+# noinspection PyRedundantParentheses,PyRedundantParentheses,PyRedundantParentheses
+class TestTaskDetails():
+
+    def test_that_person_can_view_task_details(self, base_url, selenium):
+        # This test should be non-destructive, so it cannot log in to the site
+        home_page = HomePage(base_url, selenium).open()
+        assert not home_page.is_user_logged_in
+
+        # Click the first task in the Suggested Tasks list
+        task = home_page.suggested_first_tasks[0]
+        task_name = task.name
+        task_details = task.click()
+        assert not task_details.is_user_logged_in
+
+        # Verify that the browser is on the Task Detail page for that task
+        assert task_details.is_the_current_page
```

Comment:
We don't need to assert that we're on the current page after navigating because `is_the_current_page` is called by `wait_for_page_to_load` which is called when `task.click()` is executed above.